README.md

```diff
@@ -835,6 +835,12 @@ ng-admin has unit tests (powered by karma) and end to end tests (powered by prot
 make test
 ```
 
+**Tip:** if you are working on Karma tests, you can prevent from relaunching the whole process by disabling single-run:
+
+```
+KARMA_SINGLE_RUN=false grunt karma:unit
```

Comment:
It has already been changed. This is not native to Karma, this is specific to `ng-admin`. See [Gruntfile changes](https://github.com/marmelab/ng-admin/pull/505/files#diff-35b4a816e0441e6a375cd925af50752cR53).